code/modules/mob/living/carbon/human/human.dm

```diff
@@ -392,7 +392,7 @@
 				return
 
 	//SKYRAT EDIT ADDITION BEGIN - VIEW RECORDS
-	if (is_special_character(usr))
+	if (is_special_character(usr) || usr.mind.has_exploitables_override)
```

Comment:
remove all special character stuff and just give `has_exploitable_override` to all antagonists instead.